coffee_cmd/src/main.rs

```diff
@@ -98,6 +98,23 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Nurse {} => {
+            let mut spinner = term::spinner(&format!("Trying restoring from a corrupted status!"));
+            match coffee.nurse().await {
+                Ok(val) => {
+                    let status = val.status;
+                    if status == "corrupt" {
```

Comment:
```suggestion
                    let status = val.status;
                    if status == "corrupt" {
```

It is fine, but the Rust way of doing this (also the functional way to do this, not just in Rust) is to define a type and match on this type.

So let's imagine having an enum like the following one in our model

```rust
pub enum NurseStatus {
      Currupted,
     Sane,
}
```

You than you do the match like 

```rust
match val.status {
    NurseStatus::Corrupted => {},
    NurseStatus::Sane => {},
}
```

This improves the code a lot

Reply:
Yes this is much cleaner